### PR TITLE
Fixing txinfo

### PR DESCRIPTION
**Pull request by Yhtiyar**

### CODE REVIEW

File: backend/moonstream/data.py [hunk 1]
```diff
@@ -94,8 +94,8 @@ class EthereumTransaction(BaseModel):
     gas: int
     gasPrice: int
     value: int
-    from_address: str = Field(alias="from")
-    to_address: Optional[str] = Field(default=None, alias="to")
+    from_address: str = Field(alias="from_address")
```
Comment (zomglings) [inline backend/moonstream/data.py:97]: @Yhtiyar : You don't need to put `alias="from_address"` if you are just using the same name as the member.

Why do you want to remove the alias? The alias allowed us to respect web3 schema for transactions.

File: backend/moonstream/data.py [hunk 2]
```diff
@@ -94,8 +94,8 @@ class EthereumTransaction(BaseModel):
...
+    to_address: Optional[str] = Field(default=None, alias="to_adress")
```
Comment (zomglings) [inline backend/moonstream/data.py:98]: Typo in alias - `to_adress` -> `to_address`. Question about alias still remains.

**Review by zomglings — COMMENTED** [review]
Just a small question about API request schema.

File: backend/moonstream/data.py (hunk 1 above)
Comment (Yhtiyar) [reply backend/moonstream/data.py]: Because transaction object in frontend has `from_address` instead of `from`

**Review by zomglings — APPROVED** [review]
lgtm

**Pull request merged**